### DISCUSSION TITLE
What is logged in Security Onion?

### DISCUSSION THREAD

**Discussion by IzacFrank** [Security-Onion-Solutions/securityonion#6273] [Unsupported Versions] [answered]
Hello all,

I started to use Security Onion. Maybe it is a stupid question but I’ve been asked and I need to be sure. What does Security Onion log? Only irregularities or all traffic?

Thank you.
Isac

**Comment by lock-wire** [comment, marked as answer]
Isac,
  This page from the documentation might help, https://docs.securityonion.net/en/latest/introduction.html#network-security-monitoring.  The short answer is NIDS alerts, packet metadata, full packet capture, endpoint logs from Security Onion devices, and endpoint logs ingested from devices.

Hope that helps,
Pete

**Reply by IzacFrank** [reply]
Thank you @lock-wire 
Can we say that all metadata and NIDS monitoring are logged to Suricata and Zeek logs by default? I mean not just detections.

Thank you very much again.

Cheers,
Isac

**Reply by InfosecGoon** [reply]
By default, live network traffic captured by a monitoring interface is processed in three ways:

* Full packet capture, written to disk for later retrieval.
* Metadata, generated by either Zeek or Suricata and recorded in Elasticsearch.
* NIDS alerts for any traffic that matches a Suricata IDS rules.

By default, it records everything, not just detections or alerts.

--Matt
